modules/wildfly/src/main/modules/ogm/infinispan/module.xml

```diff
@@ -10,11 +10,10 @@
         <resource-root path="hibernate-ogm-infinispan-${project.version}.jar" />
     </resources>
     <dependencies>
-        <module name="org.hibernate" export="true" services="import" slot="${hibernate.ogm.module.slot}" />
+        <module name="org.hibernate" slot="${hibernate.ogm.module.slot}" />
         <module name="org.hibernate.hql" slot="${hibernate.hql.module.slot}" />
 
-        <module name="org.infinispan" optional="true" slot="${infinispan.module.slot}" />
-        <module name="org.infinispan.commons" optional="true" slot="${infinispan.module.slot}" />
+        <module name="org.infinispan" slot="${infinispan.module.slot}" />
```

Comment:
> What exactly DYM by "resources" here?

Sorry meant services. The serviceloader definitions are resources, so the option actually exposes all resources from META-INF